lib/util/mail.php

```diff
@@ -83,14 +79,12 @@ function mail_ses($to, $subject = '(No subject)', $message = '')
         $promise = $pool->promise();
         // Force the pool to complete synchronously
         $promise->wait();
-        // $timeEnd = microtime(true);
-        // echo sprintf('Operation completed in %s seconds' . PHP_EOL, $timeEnd - $timeStart);
+
         return true;
     } catch (Exception $e) {
-        // echo sprintf('Error: %s' . PHP_EOL, $e->getMessage());
```

Comment:
Is there a reason why we're getting rid of logging here? Sounds counterproductive to me.

Reply:
I'm all for removing the commented `echo` statement, though.